### PR TITLE
MINOR: Add connector configs to site-docs

### PR DESCRIPTION
**Pull request by wicknicks**
In AK's documentation, the config props for connectors are not listed (https://kafka.apache.org/documentation/#connectconfigs). This PR adds these sink and source connector configs to the html site-docs. 

Signed-off-by: Arjun Satish <arjun@confluent.io>

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by wicknicks** [comment]
@ewencp @rhauch any thoughts on this?

File: clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java [hunk 1]
```diff
@@ -973,6 +973,19 @@ public void ensureValid(String name, Object value) {
                 validator.ensureValid(name, value);
             }
         }
+
+        @Override
+        public String toString() {
```
Comment (ewencp) [inline clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java:978]: why this override? we haven't needed something like this for the other autogenerated files

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java [hunk 2]
```diff
@@ -169,6 +169,11 @@ public void ensureValid(String name, Object value) {
                             throw new ConfigException(name, value, "Duplicate alias provided.");
                         }
                     }
+
+                    @Override
```
Comment (ewencp) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java:173]: same question about override here

**Review by ewencp — COMMENTED** [review]
couple of minor questions about overrides, but generally looks good.

File: clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java (hunk 1 above)
Comment (wicknicks) [reply clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java:978]: this Validator is used by the transforms config here: https://github.com/apache/kafka/blob/trunk/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java#L164. Without this override, the valid values column for `transforms` shows up as `org.apache.kafka.common.config.ConfigDef$CompositeValidator@736e9adb` or similar in the generated docs.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java (hunk 2 above)
Comment (wicknicks) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java:173]: same reason as above. the `CompositeValidator` is composed of a `NonNullValidator` and this anonymous validator.

File: clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java (hunk 1 above)
Comment (ewencp) [reply clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java:978]: ah, i missed that it was in the Validator and not ConfigDef itself.

**Review by ewencp — APPROVED** [review]
LGTM, thanks for the improvement!

**Comment by wicknicks** [comment]
Thanks, Ewen!

**Pull request closed without merging**